### PR TITLE
preparation work in running locally chapter

### PR DESCRIPTION
**Pull request by leolumicrosoft**
1. how to create crd before running locally
2. mention creating UDS directory before running agent locally

### CODE REVIEW

File: docs/development/development.md [hunk 1]
```diff
@@ -65,6 +65,12 @@ cargo version
     > Note: To test a specific component, use the `-p` parameter along with the [workspace member](https://github.com/project-akri/akri/blob/main/Cargo.toml). For example, to only test the Agent, run `cargo test -p agent`
 
 ## Running locally
+Before running Akri agent or controller locally, please ensure Akri configuration and instance CRDs are created in cluster, otherwise use the below command to create.
+```sh
+    kubectl apply -f ~/akri/deployment/helm/crds/akri-configuration-crd.yaml
```
Comment (kate-goldenring) [inline docs/development/development.md:70]: This path assumes that the `akri` repo is right under the home directory. I would change this to just `akri/deployment...` as in other parts of the development docs. 

Also some nits:
- can we change "created in" to "applied to" (and "to create" to "to apply them") to match the cli command?
- change to "please ensure the Akri..."

File: docs/development/development.md [hunk 2]
```diff
@@ -90,6 +96,9 @@ To locally run Akri's Agent, Controller, and Discovery Handlers as part of a Kub
     cd akri/agent
     sudo -E DEBUG_ECHO_INSTANCES_SHARED=true ENABLE_DEBUG_ECHO=1 RUST_LOG=info METRICS_PORT=8082 KUBECONFIG=$HOME/.kube/config DISCOVERY_HANDLERS_DIRECTORY=~/tmp/akri AGENT_NODE_NAME=myNode HOST_CRICTL_PATH=/usr/bin/crictl HOST_RUNTIME_ENDPOINT=/var/run/dockershim.sock HOST_IMAGE_ENDPOINT=/var/run/dockershim.sock $HOME/.cargo/bin/cargo run
     ```
+    > Note: `DISCOVERY_HANDLERS_DIRECTORY` is where Akri agent creates an unix domain socket for discovery handler's registeration. This example uses /tmp/akri that should exist or is created before executing this command.
```
Comment (kate-goldenring) [inline docs/development/development.md:99]: Can you add a new line before the note? I believe our CMS for docs needs the notes to be isolated with newlines. Also unnecessary extra newline seems to be added after.

**Review by kate-goldenring — CHANGES_REQUESTED** [review]
Thanks for adding this. Just a few modifications for clarity.

File: docs/development/development.md (hunk 1 above)
Comment (leolumicrosoft) [reply docs/development/development.md:70]: thanks, I will comply

File: docs/development/development.md (hunk 2 above)
Comment (leolumicrosoft) [reply docs/development/development.md:99]: Thanks. I will format accordingly. Btw, I am not sure if it is my fault or there is other missing step in this chapter. Right now my locally running debug echo discovery handler does not really successfully register to also locally running Akri agent, I explain in the issue report. Appreciate much if you can help.

**Review by kate-goldenring — APPROVED** [review]

**Comment by kate-goldenring** [comment]
Thank you @leolumicrosoft! I'm sure this will speed up other's development process!

**Pull request merged**